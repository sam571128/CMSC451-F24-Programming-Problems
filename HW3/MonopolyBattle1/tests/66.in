7 49 1000000000
1 1
2 3
3 3
6 3
5 3
7 5
6 6
1 7
1 6
3 6
3 5
5 2
3 7
2 4
7 6
6 4
4 1
7 4
5 6
4 2
4 5
7 2
3 2
2 7
4 4
2 2
2 1
2 5
6 5
6 2
5 4
7 1
7 7
1 5
2 6
1 2
3 1
4 7
1 4
3 4
5 5
5 1
4 3
5 7
6 1
4 6
6 7
7 3
1 3
