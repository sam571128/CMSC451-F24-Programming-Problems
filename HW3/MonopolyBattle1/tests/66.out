9935 9935 9935 9935 9935 9935 9935 